dev/com.ibm.ws.cdi.third.party_fat/publish/servers/cdi20HibernateSearchServer/persistence.xml

```diff
@@ -33,7 +33,7 @@
           value="filesystem"/>
 
 <property name="hibernate.search.default.indexBase"
-          value="/tmp/lucene/indexes"/>
+          value="SED-STRING"/>
```

Comment:
Please change the placeholder name. Something like `INDEX-PATH` would better communicate what's going to be put in here.